File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        int a[26]={0},b[26]={0};
        
        int m=s.length(),n=t.length();
        
        int i,j;
        
        for(i=0;i<m;i++) a[s[i]-'a']++;
        for(i=0;i<n;i++) b[t[i]-'a']++;
        
        int ans=0;
        
        for(i=0;i<26;i++)
        {
            ans+=abs(a[i]-b[i]);
        }
        
        return ans/2;
    }
};

File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int ans = 0;
        
        int[] freqFirst = new int[26];
        int[] freqSecond = new int[26];
        
        for(int i=0;i<s.length();i++)
        {
            int idx = (int)s.charAt(i)-(int)'a';
            freqFirst[idx]++;
        }
        
        for(int i=0;i<t.length();i++)
        {
            int idx = t.charAt(i)-'a';
            freqSecond[idx]++;
        }
        
        for(int i=0;i<26;i++)
        {
            ans += Math.abs(freqFirst[i]-freqSecond[i]);
        }
        
        ans/=2;
        return ans;
    }
}